HW_3/templates/register.html

```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Регистрация</title>
+</head>
+<body>
+    <h1>Регистрация</h1>
+    {% with messages = get_flashed_messages() %}
+        {% if messages %}
+            <ul>
+                {% for message in messages %}
+                    <li>{{ message }}</li>
+                {% endfor %}
+            </ul>
+        {% endif %}
+    {% endwith %}
+    <form method="POST" action="{{ url_for('register') }}">
+        {{ form.csrf_token }}
+        <label for="first_name">Имя:</label>
+        {{ form.first_name() }}
+        <br>
+        <label for="last_name">Фамилия:</label>
+        {{ form.last_name() }}
+        <br>
+        <label for="email">Email:</label>
+        {{ form.email() }}
+        <br>
+        <label for="password">Пароль:</label>
+        {{ form.password() }}
+        <br>
+        {{ form.submit() }}
```

Comment:
А что если форма будет иметь более 5 полей? Лучше сделать автоматизацию
```suggestion
    {% for obj in form %}
    {% if obj.name == "csrf_token" %}
    {{ obj }}
    {% else %}
    <p>{{ obj.label }}: {{ obj }}</p>
            {% for error in obj.errors %}
                <span style="color: red;">{{ error }}</span><br>
            {% endfor %}
    {% endif %}
    {% endfor %}
```
Причем строки
```html
            {% for error in obj.errors %}
                <span style="color: red;">{{ error }}</span><br>
            {% endfor %}
```
будут повторять flash сообщение об ошибке. Тут вы можете решить, что лучше: либо выводить сообщение тут, либо выше.